File: year2/MPP/Java Client Server Sockets/SablonJavaDB/src/main/java/me/teledonServices/ITeledonObserver.java
package me.teledonServices;

import me.networking.dto.CazCaritabilDTO;

import java.util.List;

public interface ITeledonObserver {
    void donationMade(List<String> donatori, List<CazCaritabilDTO> cazuri) throws TeledonExeption;
    void commitDonatori(List<String> donatori) throws TeledonExeption;
}

File: year2/semester2/MPP/C# Client Server Sockets/Proiect_MPP_C#_VARIANTA_FINALA/servicess/ITeledonObserver.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using DomainRepoLibrary.entities;

namespace servicess
{
    public interface ITeledonObserver
    {
        void donationMade(List<CazDTO> cazuri);
        void commitDonatori(List<string> donatori);
    }
}
